src/spark/utils.py

```diff
@@ -99,7 +83,7 @@ def get_spark_session(
         yarn: bool = True,
         delta_lake: bool = True,
         executor_cores: int = DEFAULT_EXECUTOR_CORES,
-        scheduler_pool: str = "default") -> SparkSession:
+        scheduler_pool: str = SPARK_DEFAULT_POOL) -> SparkSession:
```

Comment:
This method is now ~100 lines long, which is really long

Reply:
yea. I know. It's probably okay for now. It's break down with different configurations pretty clearly.